src/protocols/gwmp/hpr_gwmp_worker.erl

```diff
@@ -237,7 +239,7 @@ handle_udp(
                         State0;
                     {DestMap1, Resend} ->
                         {{A, B, C, D}, Port} = DataSrc,
-                        Key = {lists:flatten(io_lib:format("~p.~p.~p.~p", [A, B, C, D])), Port},
+                        Key = {{A, B, C, D}, Port},
```

Comment:
Isn't this `Key = DataSrc`? Do we need `Key`?

Reply:
The address pattern match only matches ipv4 addresses. Do we care about ipv6 addresses?

Reply:
That it is. It is not needed.

I think that restriction only applies to this branch where we're remapping a destination based on region. 
I'm hoping we can drop this code before long in lieu of a better solution.

If it's easy enough to support ipv6 here, and it won't fail silently, by all means let's do that.